danielMunteanu/src/main/java/danielCollectionTask/Item.java

```diff
@@ -0,0 +1,57 @@
+package danielCollectionTask;
+
+import java.util.UUID;
+
+public class Item {
+    private final String id;
+    private String title;
+    private Type type;
+    private double price;
+
+    public Item(String title, Type type, double price) {
+        this.id = UUID.randomUUID().toString();
+        this.title = title;
+        this.type = type;
+        this.price = price;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public Type getType() {
+        return type;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public void setPrice(double price) {
```

Comment:
price can never be negative, can you please handle this situation by using if()?

Reply:
done